Index: src/main/java/com/jkxy/car/api/utils/JSONResult.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/jkxy/car/api/utils/JSONResult.java	(date 1666159045000)
+++ src/main/java/com/jkxy/car/api/utils/JSONResult.java	(date 1666159113549)
@@ -18,6 +18,7 @@
     private String msg;
 
     // 响应中的数据
+
     private Object data;
 
 
